cartographer_grpc/framework/server.cc

```diff
@@ -41,8 +45,13 @@ Server::Server(const Options& options) : options_(options) {
   server_builder_.AddListeningPort(options_.server_address,
                                    grpc::InsecureServerCredentials());
 
+  // Set up event queue threads.
+  for (size_t i = 0; i < options_.num_event_threads; ++i) {
```

Comment:
Just use the constructor to create n copies:
event_queue_threads_ = vector<...>(num_event_threads);

Reply:
Done.